README.md

```diff
@@ -5,15 +5,15 @@ Crosswalk for iOS is a sub-project of [Crosswalk](https://crosswalk-project.org/
 
 * Extended WKWebView
 
-  Crosswalk for iOS build on top of `WKWebView`, the mordern WebKit framework debuted in iOS 8.0. We extend the WKWebView to build Crosswalk extension framework within, and we are intended to bring the least intrusion to the interface to make developers use it as natural as WKWebView itself. For the detailed information you may refer to [Embedding Mode & WKWebView](https://github.com/otcshare/crosswalk-ios/wiki/Embedding-Mode-&-Native-APIs).
+  Crosswalk for iOS build on top of `WKWebView`, the mordern WebKit framework debuted in iOS 8.0. We extend the WKWebView to build Crosswalk extension framework within, and we are intended to bring the least intrusion to the interface to make developers use it as natural as WKWebView itself. For the detailed information you may refer to [Embedding Mode & WKWebView](https://github.com/crosswalk-project/crosswalk-ios/wiki/Embedding-Mode-&-Native-APIs).
```

Comment:
I think
"Crosswalk for iOS build on top of"
should be
"Crosswalk for iOS is built on top of"